packages/@/@fontsource/abeezee/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@fontsource/abeezee",
+  "description": "Self-host the ABeeZee font in a neatly bundled NPM package.",
+  "main": "index.css",
+  "keywords": [
+    "fontsource",
+    "font",
+    "font family",
+    "google fonts",
+    "abeezee",
+    "ABeeZee",
+    "css",
+    "sass",
+    "front-end",
+    "web",
+    "typeface",
+    "variable"
+  ],
+  "author": "Google Inc.",
+  "license": "OFL-1.1",
+  "homepage": "https://fontsource.org/fonts/abeezee",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fontsource/font-files.git",
+    "directory": "fonts/google/abeezee"
+  },
+  "autoupdate": {
+    "excludeFromSearch": true,
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*"
```

```suggestion
          "*.css",
          "files/*.@(woff|woff2)"
```
Looking at the package structure, we only need the CSS files from the root, and the font files themselves are within `files`